tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -122,3 +122,20 @@ Feature: State of the quota
       | /filesForUpload/lorem-big.txt | /ocs/v1.php/cloud/users/%username% | 100      | 91.17          |
       | /filesForUpload/lorem.txt     | /ocs/v2.php/cloud/users/%username% | 200      | 6.99           |
       | /filesForUpload/lorem-big.txt | /ocs/v2.php/cloud/users/%username% | 200      | 91.17          |
+
+
+  @env-config
+  Scenario: upload a file by setting OCIS spaces max quota
+    Given the config "OCIS_SPACES_MAX_QUOTA" has been set to "10"
+    And user "Brian" has been created with default attributes and without skeleton files
+    When user "Brian" uploads file with content "more than 10 bytes content" to "lorem.txt" using the WebDAV API
+    Then the HTTP status code should be "507"
+
+
+  @env-config
+  Scenario: create a space by setting OCIS spaces max quota
+    Given the config "OCIS_SPACES_MAX_QUOTA" has been set to "50"
+    And user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has given "Brian" the role "Space Admin" using the settings api
+    When user "Brian" creates a space "new space" of type "project" with quota "51" using the Graph API
+    Then the HTTP status code should be "500"
```

Comment:
@ScharfViktor @micbar is this supposed to be 500? or might be 507?

Reply:
Issue created here https://github.com/owncloud/ocis/issues/6414

Reply:
@SagarGi can we add the passing scenario for now and add this failing scenario in the next PR?

Reply:
i will remove the doubtful scenarios from this PR and create another PR since we can use the steps for other scenarios.